asciidocs/modules.adoc

```diff
@@ -199,7 +199,7 @@ workflow  {
 }
 ----
 
-If a process defines two or more output channels, each channel can be accessed by indexing the `.out` attribute, e.g., `.out[0]`, `.out[1]`, etc. In our example we only have the `[0]`'th output:
+If a process defines two or more output channels, each channel can be accessed by indexing the `.out` attribute, e.g., `.out[0]`, `.out[1]`, etc. In our example we only have the `[0]'th` output:
```

Comment:
Slightly controversial, but I'm happy if there's a reason (eg. it's easier to read this way)...

Reply:
Before, it rendered like this 😅 
<img width="943" alt="Screen Shot 2022-09-19 at 02 00 20" src="https://user-images.githubusercontent.com/1023197/190952425-5fd8f1c1-e08a-4094-963f-b9a774d771be.png">
Now:
<img width="978" alt="Screen Shot 2022-09-19 at 02 01 06" src="https://user-images.githubusercontent.com/1023197/190952473-0e1af9e7-1c2a-4ca7-96f8-b50e801c60ad.png">

I'm fine if you want to suggest some other way. I just found a similar way that rendered correctly 😄

Reply:
Then yeah, that's a good reason :)